crates/ty_project/src/db/changes.rs

```diff
@@ -42,6 +45,16 @@ impl ProjectDatabase {
             tracing::trace!("Handle change: {:?}", change);
 
             if let Some(path) = change.system_path() {
+                if let Some(config_file) = &config_file_override {
+                    if let Ok(config_file) = self.system().canonicalize_path(config_file) {
```

Comment:
Is it necessary that we canonicalize the path here or could we avoid it?

Reply:
@MichaReiser makes sense - thanks! this is ready for review